### PR TITLE
fix missing spaces

### PR DESCRIPTION
**Pull request by ElevenTom**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
31 words might be misspelled, please check them: aallier, appendChild, attachShadow, attributeChangedCallback, Compponent, connectedCallback, css, CSS, customELement, customElements, d'ElementName, developpers, d'HTMLElement, disconnectedCallback, disparaitre, HTMLElement, imperatif, installable, lang, l’HTML, MDN, polyfill, Polyfill, popin, popinTemplate, précédement, shadowDOM, shadowRoot, WebComponent, webComponents, WebComponents

File: _posts/fr/2020-07-15-webcomponent.md [hunk 1]
```diff
@@ -40,7 +40,9 @@ Concrètement ça ressemble à ça :
 
 Les balises `template` ne sont pas affichées sur la page, elles doivent être copiées puis ajoutées au DOM grâce à `appendChild`.
 La balise `slot` permet d'insérer du HTML à la place grâce a la propriété `name`
-###customELement
+
+### customELement
```
Comment (VEBERArnaud) [inline _posts/fr/2020-07-15-webcomponent.md:44]:
```suggestion
### customElement
```

File: _posts/fr/2020-07-15-webcomponent.md [hunk 2]
```diff
@@ -40,7 +40,9 @@ Concrètement ça ressemble à ça :
...
+
```
Comment (VEBERArnaud) [inline _posts/fr/2020-07-15-webcomponent.md:45]:
```suggestion
```

**Comment by ElevenWilson** [comment]
31 words might be misspelled, please check them: aallier, appendChild, attachShadow, attributeChangedCallback, Compponent, connectedCallback, css, CSS, customElement, customElements, d'ElementName, developpers, d'HTMLElement, disconnectedCallback, disparaitre, HTMLElement, imperatif, installable, lang, l’HTML, MDN, polyfill, Polyfill, popin, popinTemplate, précédement, shadowDOM, shadowRoot, WebComponent, webComponents, WebComponents

**Pull request closed without merging**